bootstrap/roles/ocp4-install-gitops/tasks/gitops.yaml

```diff
@@ -53,35 +53,39 @@
     state: present
     definition: "{{ lookup('template', 'subs-pipelines.yml.j2') }}"
 
-- name: Adapt to the openshift_cluster_version LESS than 4.9
-  when: ocp4_cluster_version is version_compare('4.9', '<')
-  block:
-
-  - name: Wait for Pipelines CRD to exist
-    kubernetes.core.k8s_info:
-      api_version: "apiextensions.k8s.io/v1beta1"
-      kind: CustomResourceDefinition
-      name: "{{ item }}"
-    loop: "{{ pipelines_expected_crds }}"
-    register: crds
-    until: crds.resources|length > 0
-    retries: 30
-    delay: 10
-
-- name: Adapt to the openshift_cluster_version MORE than 4.9
-  when: ocp4_cluster_version is version_compare('4.9', '>=')
-  block:
-
-  - name: Wait for Pipelines CRD to exist
-    kubernetes.core.k8s_info:
-      api_version: "apiextensions.k8s.io/v1"
-      kind: CustomResourceDefinition
-      name: "{{ item }}"
-    loop: "{{ pipelines_expected_crds }}"
-    register: crds
-    until: crds.resources|length > 0
-    retries: 30
-    delay: 10
+# - name: Adapt to the openshift_cluster_version LESS than 4.9
```

Comment:
@afouladi7 why we are removing these checks? I've tested in 4.13 and worked like a charm.

Reply:
Are you running this on BM? I tested this on both BM and RHPDS on 4.12.

Reply:
VMs (Hetzner + libvirt + 4.13) and I'll do it in RHPDS 4.11 as well :)